### PR TITLE
Medical GUI - Improve Selected Category colouration

### PR DESCRIPTION
**Pull request by SpicyBagpipes**
**When merged this pull request will:**
- Add colouration for currently selected category.
- Add slight transparency increase when mousing over category

### IMPORTANT

- [x] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by SpicyBagpipes** [comment]
Unsure why validate failed, am using this without issue currently and cannot find any syntax errors

File: addons/medical_gui/functions/fnc_updateCategories.sqf [hunk 1]
```diff
@@ -24,8 +24,17 @@ params ["_display"];
     private _enable = GVAR(actions) findIf {_category == _x select 1 && {call (_x select 2)}} > -1;
     _ctrl ctrlEnable _enable;
 
+    private _selectedColor = [
+        profilenamespace getvariable ["GUI_BCG_RGB_R",0.13],
+        profilenamespace getvariable ["GUI_BCG_RGB_G",0.54],
+        profilenamespace getvariable ["GUI_BCG_RGB_B",0.21],
+        profilenamespace getvariable ["GUI_BCG_RGB_A",0.8]
```
Comment (jonpas) [inline addons/medical_gui/functions/fnc_updateCategories.sqf:31]:
```suggestion
        profileNamespace getVariable ["GUI_BCG_RGB_R", 0.13],
        profileNamespace getVariable ["GUI_BCG_RGB_G", 0.54],
        profileNamespace getVariable ["GUI_BCG_RGB_B", 0.21],
        profileNamespace getVariable ["GUI_BCG_RGB_A", 0.8]
```

File: addons/medical_gui/functions/fnc_updateCategories.sqf [hunk 2]
```diff
@@ -24,8 +24,17 @@ params ["_display"];
...
+    ];
     private _color = [[0.4, 0.4, 0.4, 1], [1, 1, 1, 1]] select _enable;
+    _color = if (GVAR(selectedCategory) isEqualTo _category) then {_selectedColor} else {_color};
     _ctrl ctrlSetTextColor _color;
+    _color set [-1, 0.8]; //Mouseover change
```
Comment (jonpas) [inline addons/medical_gui/functions/fnc_updateCategories.sqf:36]:
```suggestion
    _color set [-1, 0.8]; // Mouseover change
```

File: addons/medical_gui/functions/fnc_updateCategories.sqf [hunk 3]
```diff
@@ -24,8 +24,17 @@ params ["_display"];
...
     _ctrl ctrlEnable _enable;
 
+    private _selectedColor = [
+        profilenamespace getvariable ["GUI_BCG_RGB_R",0.13],
+        profilenamespace getvariable ["GUI_BCG_RGB_G",0.54],
+        profilenamespace getvariable ["GUI_BCG_RGB_B",0.21],
+        profilenamespace getvariable ["GUI_BCG_RGB_A",0.8]
+    ];
     private _color = [[0.4, 0.4, 0.4, 1], [1, 1, 1, 1]] select _enable;
+    _color = if (GVAR(selectedCategory) isEqualTo _category) then {_selectedColor} else {_color};
```
Comment (jonpas) [inline addons/medical_gui/functions/fnc_updateCategories.sqf:34]:
```suggestion
    _color = [_color, _selectedColor] select (GVAR(selectedCategory) isEqualTo _category);
```

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/medical_gui/functions/fnc_updateCategories.sqf (hunk 3 above)
Comment (SpicyBagpipes) [reply addons/medical_gui/functions/fnc_updateCategories.sqf:34]: Knew there was a better way to do it, thanks

**Review by LinkIsGrim — APPROVED** [review]

File: addons/medical_gui/gui.hpp [hunk 4]
```diff
@@ -224,6 +224,7 @@ class ACE_Medical_Menu {
             w = QUOTE(POS_W(1.5));
             h = QUOTE(POS_H(1.5));
             color[] = {1, 1, 1, 1};
+            colorActive[] = {1,1,1,0.8};
```
Comment (jonpas) [inline addons/medical_gui/gui.hpp:227]:
```suggestion
            colorActive[] = {1, 1, 1, 0.8};
```

Comment (SpicyBagpipes) [reply addons/medical_gui/gui.hpp]: Had already pushed an update to change this, guess it didn't go through

**Review by jonpas — APPROVED** [review]

**Pull request merged**